website/package-lock.json

```diff
@@ -445,7 +445,6 @@
       "integrity": "sha512-5dgndWOriYSm5cnYaJNhalLNDKOqFwyDB/rr1E9ZsGciGvKPs8R2xYGCacuf3z6K1YKDz182fd+fY3cn3pMqXQ==",
       "dev": true,
       "dependencies": {
-        "graceful-fs": "^4.1.6",
```

Comment:
Argh sorry, not related. I've been having this line change every time I preview the documentation locally. It finally managed to find its way into a commit, my bad. Let's ignore this.